Automattic/wp-calypso
# Signup: Update website type icons in the signup flow.

## How to test

You should be able to see the new website type icons when you open `/start/design-type`(with login) or `/start/design-type-with-store`(without login).

Expected:
![screen shot 2017-09-06 at 21 18 56](https://user-images.githubusercontent.com/212034/30154798-f390d8a8-93f5-11e7-832e-175a50c1eec1.png)
![screen shot 2017-09-06 at 21 19 10](https://user-images.githubusercontent.com/212034/30154799-f3944cea-93f5-11e7-95f3-0640b79640b7.png)

This PR is a part of #14863.

cc @cristelrossignol @markryall @fditrapani
